Bad import using --baseQuery when generated file and baseQuery file are in the same folder

**Description**
Running
```shell
npx @rtk-incubator/rtk-query-codegen-openapi \
--file ./src/services/api.generated.ts \
--baseQuery ./src/services/custom-fetch-base-query.ts:customFetchBaseQuery \
--hooks \
../django/openapi-schema.yml
```
The import string at the top of `api.generated.ts` is `import { customFetchBaseQuery } from "custom-fetch-base-query";` resulting in error `TS2307: Cannot find module 'custom-fetch-base-query' or its corresponding type declarations`. 
`api.generated.ts` and `custom-fetch-base-query.ts` are in the same folder.

**Resolution**
The import string should be `import { customFetchBaseQuery } from "./custom-fetch-base-query";`

--- a/src/utils/resolveImportPath.ts
+++ b/src/utils/resolveImportPath.ts
@@ -4,7 +4,7 @@ import { stripFileExtension } from './stripFileExtension';
 export function resolveImportPath(modulePath: string, containingFile: string) {
   containingFile = path.resolve(containingFile, '..');
   const strippedFile = stripFileExtension(path.relative(containingFile, modulePath));
-  if (strippedFile[0] !== '.') {
+  if (strippedFile.charAt(0) !== '.') {
     return `./${strippedFile}`;
   }
   return strippedFile;

--- a/src/utils/resolveImportPath.ts
+++ b/src/utils/resolveImportPath.ts
@@ -3,5 +3,9 @@ import { stripFileExtension } from './stripFileExtension';
 
 export function resolveImportPath(modulePath: string, containingFile: string) {
   containingFile = path.resolve(containingFile, '..');
-  return stripFileExtension(path.relative(containingFile, modulePath));
+  const strippedFile = stripFileExtension(path.relative(containingFile, modulePath));
+  if (strippedFile[0] !== '.') {
+    return `./${strippedFile}`;
+  }
+  return strippedFile;
 }

--- a/test/cli.test.ts
+++ b/test/cli.test.ts
@@ -12,6 +12,13 @@ function getTmpFileName() {
   return path.resolve(tmpDir, `${++id}.test.generated.ts`);
 }
 
+function copyAndGetTmpFileName(fileToCopyPath: string, newFileName: string): string {
+  newFileName = `${++id}.test.${newFileName}`;
+  const newFilePath = path.resolve(tmpDir, newFileName);
+  fs.copyFileSync(fileToCopyPath, newFilePath, fs.constants.COPYFILE_EXCL);
+  return newFileName;
+}
+
 function cli(args: string[], cwd: string): Promise<{ error: ExecException | null; stdout: string; stderr: string }> {
   return new Promise((resolve) => {
     exec(
@@ -164,6 +171,34 @@ describe('CLI options testing', () => {
 
     expect(result.stdout).not.toContain('fetchBaseQuery');
     expect(result.stdout).toContain(`import { default as customBaseQuery } from \"test/fixtures/customBaseQuery\"`);
+
+    const expectedErrors = [MESSAGES.NAMED_EXPORT_MISSING];
+
+    const numberOfErrors = expectedErrors.filter((msg) => result.stderr.indexOf(msg) > -1).length;
+    expect(numberOfErrors).toEqual(0);
+  });
+
+  it("should import { default as customBaseQuery } from './customBaseQuery' when a local customBaseQuery is provided to --baseQuery", async () => {
+    const localBaseQueryName = copyAndGetTmpFileName('./test/fixtures/customBaseQuery.ts', 'localCustomBaseQuery.ts');
+    const fileName = getTmpFileName();
+    const result = await cli(
+      [
+        '-h',
+        `--baseQuery`,
+        `./test/tmp/${localBaseQueryName}:namedBaseQuery`,
+        '--file',
+        fileName,
+        `./test/fixtures/petstore.json`,
+      ],
+      '.'
+    );
+
+    const output = fs.readFileSync(fileName, { encoding: 'utf-8' });
+
+    const strippedLocalBaseQueryName = path.parse(localBaseQueryName).name;
+
+    expect(output).not.toContain('fetchBaseQuery');
+    expect(output).toContain(`import { namedBaseQuery } from './${strippedLocalBaseQueryName}'`);
 
     const expectedErrors = [MESSAGES.NAMED_EXPORT_MISSING];
 

